src/Integration.Vsix/Commands/ConnectedModeMenu/ConnectedModeMenu.vsct

```diff
@@ -8,11 +8,32 @@
       </Group>
     </Groups>
 
+    <Buttons>
```

Comment:
As agreed with Marco, the new command is only visible in the SonarLint menu, so there is no need for command placement like the original command had